docs/topics/vscode-extension-resolving-issues.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-resolving-issues_{context}']
+= Resolving issues
+
+You can resolve issues by using a Quick Fix (automatic code replacement) or by editing the code.
+
+.Procedure
+
+. Resolve issues detected by the {ProductShortName} extension by performing one of the following actions:
+
+* If the issue displays the Quick Fix icon (image:vs_optional.png[Optional or Quick Fix]), right-click the issue and select *Preview Quick Fix*. To accept the suggested fix, right-click the issue and select *Apply Quick Fix.*
+* If the issue has any other icon or you decide not to use all suggested Quick Fixes, right-click the issue, select *Open Code* to open the file involved, make any changes needed to the code, and then save the file.
+
+. Optional: After you fix an issue, you can right-click it and select *Mark as Complete*. The extension displays the Resolved icon (image:vs_resolved.png[Resolved]) next to the issue. This has no effect on the code.
+
+. Optional: To remove an issue from the list, right-click it and select *Delete*. This has no effect on the code.
++
+*Delete* and *Mark as complete* are designed to make it easier for you to keep track of which issues you have finished working with and which still need attention. Files marked deleted or completed will be analyzed again the next time you analyze a project that contains them.
```

Comment:
Check the capitalization of "Mark as C/complete". You have "C" earlier in the procedure.